app/(auth)/register/RegisterForm.tsx

```diff
@@ -31,7 +31,8 @@ export default function RegisterForm(props: { returnTo?: string | string[] }) {
         const returnTo = getSafeReturnToPath(props.returnTo);
 
         if (returnTo) {
-          router.push(returnTo);
+          router.replace(returnTo);
```

Comment:
we decided we want `.push()` - adds a history state

Reply:
[`1e75560` (#13)](https://github.com/upleveled/next-js-example-winter-2023-vienna-austria/pull/13/commits/1e75560683fe5978622c38e7866916237bbbcc47)